command/agent/consul/namespaces_client.go

```diff
@@ -68,7 +68,8 @@ func (ns *NamespacesClient) allowable(now time.Time) bool {
 // List returns a list of Consul Namespaces.
 //
 // TODO(shoenig): return empty string instead of "default" when namespaces are not
-//   enabled. (Coming in followup PR).
+//
+//	enabled. (Coming in followup PR).
```

Comment:
I think that's how it should look like?

```suggestion
//  enabled. (Coming in followup PR).
```

Reply:
I'll just make the whole thing one line